tools/cwrap/plugins/NullableArguments.py

```diff
@@ -12,3 +12,16 @@ def process_single_unpack(self, code, arg, arg_accessor):
         if 'nullable' in arg and arg['nullable']:
             return '({} == Py_None ? NULL : {})'.format(arg_accessor, code)
         return code
+
+
+class UndefinedArguments(CWrapPlugin):
```

Comment:
Any reason why you can't replace `NullableArguments`? It's not used anywhere as far as I can see

Reply:
nnwrap seems to use it. In any case, UndefinedArguments is going to get deleted soon when I have ATen take over cwrap generation.